quickwit-indexing/src/actors/packager.rs

```diff
@@ -17,7 +17,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-use std::collections::HashSet;
+use std::collections::BTreeSet;
```

Comment:
this change is driven by 2 things I guess:
- it's nice to sort tags
- we do iterations of this field